package.json

```diff
@@ -4,9 +4,9 @@
   "description": "Remote new tab (third version)",
   "main": "src/main.js",
   "scripts": {
-    "setup": "npm run clean && npm run build:static",
+    "setup": "npm run clean && npm run build:static && npm run build:offline",
```

Comment:
actually. it needs to look at www. So maybe we have to do a full build during setup :/